### DISCUSSION TITLE
Styles are not applied when using preset

### DISCUSSION THREAD

**Discussion by huynq55** [system-ui/theme-ui#2073] [Q&A] [answered]
Hello, I want to ask a question. I installed gatsby-plugin-theme-ui. I created index.js in src/gatsby-plugin-theme-ui with the following content:

`import { swiss } from '@theme-ui/presets'

export default swiss`

however, it looks like the styles are not applied, for example I checked an a tag:
![Capture](https://user-images.githubusercontent.com/1425348/149052469-ee0bc526-6c64-445f-af71-bee66564582e.PNG)

**Comment by lachlanjc** [comment, marked as answer]
Hey! Is the plugin included in your `gatsby-config` file? Can you upload a repo to GitHub so folks can take a look?

**Reply by huynq55** [reply]
I think I forgot to use <Themed.a>

**Reply by hasparus** [reply]
You can use `BaseStyles` in use cases where you want to cascade all .styles to children.

```jsx
<BaseStyles>
  <h1>
    Styled based on <code>theme.styles</code>
  </h1>
</BaseStyles>
```
